packages/text-bitmap/src/BitmapFont.ts

```diff
@@ -20,13 +31,18 @@ const available = {};
  */
 export class BitmapFont
 {
-    constructor(data, textures)
+    public readonly font: string;
+    public readonly size: number;
+    public readonly lineHeight: number;
+    public chars: Dict<IBitmapFontCharacter>;
```

Comment:
JSDoc marks `chars` as read-only, but it is set to null below.

Reply:
In previous conversion we have preserved the JSDoc readonly when its set to null in `destroy` 

If you take a look inside `AbstractRender` and its `destroy` method you will see how its been handled in the past

Reply:
Oh yes I see, will do